compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -3334,6 +3372,7 @@ impl<'tcx> LayoutCx<'tcx, TyCtxt<'tcx>> {
     }
 }
 
+#[tracing::instrument(level = "debug", skip(cx))]
```

Comment:
Why do you use "debug" for all this extremely verbose logging? There is a "trace" level after all.